backend/estimate-service/src/main/java/org/egov/works/web/models/workflow/BusinessService.java

```diff
@@ -0,0 +1,100 @@
+package org.egov.works.web.models.workflow;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import lombok.*;
+import org.egov.works.web.models.AuditDetails;
+import org.springframework.validation.annotation.Validated;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A Object holds the
+ */
+@ApiModel(description = "A Object holds the")
+@Validated
+@javax.annotation.Generated(value = "org.egov.codegen.SpringBootCodegen", date = "2018-12-04T11:26:25.532+05:30")
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@ToString
+@EqualsAndHashCode(of = {"tenantId","businessService"})
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class BusinessService   {
```

Comment:
Are these files present in projects like pgr and pt? Or have they used it from some library? 
There is a new library being developed. Look if it has any reusable classes. 
[digit-models](https://github.com/egovernments/DIGIT-Dev/tree/develop/core-services/libraries/digit-models)

Reply:
done.